eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -75,6 +77,12 @@ function GetAdjustedReadmeContent($pkgInfo, $lang){
     if ($headerContentMatches) {
       $foundTitle = $headerContentMatches.Matches[0]
       $fileContent = $pkgInfo.ReadmeContent -replace $foundTitle, "$foundTitle - Version $($pkgInfo.PackageVersion) `n"
+      # Replace github master link with release tag.
+      $regex = new-object System.Text.RegularExpressions.Regex ($releaseReplaceRegex,
```

Comment:
I don't think the custom regex object is needed here.

Reply:
Answered above

Reply:
This only needs to be:
```
$fileContent = $fileContent -replace $releaseReplaceRegex, "`${1}$($pkgInfo.Tag)`$2"
```

Reply:
Just want to take the regex options. If writing like this, we are checking case as well.

Reply:
`-replace` already ignores case by default (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-7#replacement-operator) and we can also make it case sensitive if we need to by changing it to `-creplace`